seqr/management/tests/reload_clinvar_all_variants_tests.py

```diff
@@ -0,0 +1,310 @@
+                    </Classifications>
+                </ClassifiedRecord>
+            </VariationArchive>
+            '''
+        data += '</ClinVarVariationRelease>'
+        responses.add(responses.GET, WEEKLY_XML_RELEASE, status=200, body=gzip.compress(data.encode()), stream=True)
+        call_command('reload_clinvar_all_variants')
+        mock_logger.assert_called_with('Updating Clinvar ClickHouse tables to 2025-06-30 from None.')
+        self.assertEqual(ClinvarAllVariantsSnvIndel.objects.all().count(), BATCH_SIZE * 2 + 10)
+
+    @responses.activate
+    def test_malformed_variants(self, mock_logger, mock_safe_post_to_slack):
+        for description, review_status, conflicting_pathogenicities in [
+            ("Pathogenic-ey", None, None),  # Unhandled Pathogenicity
+            ("Pathogenic; but unknown assertion", None, None),  # Unhandled Assertion
+            ("Pathogenic", "unhandled", None),  # Unhandled Review Status
+            ("Conflicting classifications of pathogenicity", None, "Pathogenic;"),
+            ("Conflicting classifications of pathogenicity", None, "Pathogenic(18); unhandled(1)")
+        ]:
+            data = f'''<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+            <ClinVarVariationRelease xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                                     xsi:noNamespaceSchemaLocation="http://ftp.ncbi.nlm.nih.gov/pub/clinvar/xsd_public/ClinVar_VCV_2.4.xsd"
+                                     ReleaseDate="2025-06-30">
+                <VariationArchive>
+                    <ClassifiedRecord>
+                        <SimpleAllele AlleleID="1" VariationID="5603">
+                            <Location>
+                                <SequenceLocation Assembly="GRCh38" Chr="1" variantLength="1"
+                                                  positionVCF="1" referenceAlleleVCF="G" alternateAlleleVCF="A"/>
+                            </Location>
+                        </SimpleAllele>
+                        <Classifications>
+                            <GermlineClassification>
+                                <Description>{description}</Description>
+                                {f"<ReviewStatus>{review_status}</ReviewStatus>" if review_status else ""}
+                                {f"<Explanation>{conflicting_pathogenicities}</Explanation>" if conflicting_pathogenicities else ""}
+                            </GermlineClassification>
+                        </Classifications>
+                    </ClassifiedRecord>
+                </VariationArchive>
+            </ClinVarVariationRelease>'''
+            responses.add(
+                responses.GET,
+                WEEKLY_XML_RELEASE,
+                status=200,
+                body=gzip.compress(data.encode()),
+                stream=True,
+            )
+
+            with self.assertRaises(CommandError):
+                call_command('reload_clinvar_all_variants')
```

Comment:
we should actually test that the expected error is raised - this test would succeed if every single one of these failed on the first error and many edge cases could be missed